sirepo/package_data/static/js/raydata.js

```diff
@@ -724,6 +738,36 @@ SIREPO.app.directive('scanSelector', function() {
                 return $scope.searchForm.$dirty && $scope.searchStartTime && $scope.searchStopTime;
             };
 
+            $scope.showSelectAllButton = (index) => {
+                return index === 0;
+            };
+
+            $scope.showSortButton = (index) => {
+                return index > 0 && index === hoveredIndex;
+            };
+
+            $scope.sortCol = (column) => {
+                if (column === 'selected') {
+                    return;
+                }
+                $scope.orderByColumn = column;
+                $scope.reverseSortScans = ! $scope.reverseSortScans;
+            };
+
+            $scope.toggleSelectAll = () => {
```

Comment:
The code between the if and else is repeated. I think it could be simplified to
```js
$scope.scans.forEach((s) => {
        $scope.toggleScanSelection(
            s, 
            ! $scope.selectAllColumns,
        );
})
$scope.selectAllColumns = ! $scope.selectAllColumns;
```

Reply:
eb41875ad13fdf2a9ff2f5e261bcf5432c048ace